docs/source/api/index.md

```diff
@@ -2,24 +2,19 @@
 
 ## Top level objects
 ```{eval-rst}
-.. currentmodule:: pymmmc
+.. currentmodule:: pymc-marketing
```

Comment:
This needs to be the string used to import the package/module. Now sphinx can't import anything and the api docs are empty: https://pymc-marketing--116.org.readthedocs.build/en/116/api/index.html
I also think this section is no longer needed as there are no longer top level objects right? Only the two modules which are listed below